DevelopGitHub/GitHubExtension/GitHubExtension.Statistics.WebApi/Controllers/StatisticController.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+using System.Web.Routing;
+using GitHubExtension.Statistics.WebApi.CommunicationModels;
+using GitHubExtension.Statistics.WebApi.Constant;
+using GitHubExtension.Statistics.WebApi.Extensions.Identity;
+using GitHubExtension.Statistics.WebApi.Queries.Interfaces;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Statistics.WebApi.Controllers
+{
+    public class StatisticController : ApiController
+    {
+        private readonly IStatisticsQuery _statisticsQuery;
+
+        public StatisticController(IStatisticsQuery statisticsQuery)
+        {
+            this._statisticsQuery = statisticsQuery;
+        }
+
+        public HttpContext GetHttpContext
```

Comment:
Api controller as I remember already has property RequestContext...Please use it insterad of HttpContext.Current.